velox/expression/FunctionSignature.h

```diff
@@ -58,6 +60,27 @@ class SignatureVariable {
     return knownTypesOnly_;
   }
 
+  void setKnownTypesOnly(bool knownTypesOnly) {
```

Comment:
Let's keep this class immutable and remove these setters.

Reply:
Got it, fixed.